packages/lib/src/vis/hooks.ts

```diff
@@ -98,14 +95,9 @@ export function useCssColors(
 
 export function useHtmlCoords<T extends Vector3[]>(
   ...worldCoords: T
-): MappedTuple<T, Vector3> {
+): MappedTuple<T> {
   const { worldToHtml } = useVisCanvasContext();
-  return useCameraState(
-    (camera) =>
-      worldCoords.map((pt) => worldToHtml(camera, pt)) as MappedTuple<
-        T,
-        Vector3
```

Comment:
The second generic of `MappedTuple` is now optional: it assumes you want a tuple of the same type. This avoids this sort of ugly wrapping.